### PR TITLE
[circle-quantizer] Add profiling_data_gen option

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds profiling_data_gen option to circle-quantizer.

Related: #6080
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/circle-quantizer/src/CircleQuantizer.cpp [hunk 1]
```diff
@@ -98,6 +102,12 @@ int entry(int argc, char **argv)
   arser.add_argument("input").nargs(1).type(arser::DataType::STR).help("Input circle model");
   arser.add_argument("output").nargs(1).type(arser::DataType::STR).help("Output circle model");
 
+  arser.add_argument("--profiling_data_gen")
```
Comment (jinevening) [inline compiler/circle-quantizer/src/CircleQuantizer.cpp:105]: How about this?

```suggestion
  arser.add_argument(pdg)
```

**Review by jinevening — APPROVED** [review]
LGTM

File: compiler/circle-quantizer/src/CircleQuantizer.cpp (hunk 1 above)
Comment (mhs4670go) [reply compiler/circle-quantizer/src/CircleQuantizer.cpp:105]: @jinevening Oh, good point:)

**Review by jinevening — APPROVED** [review]
LGTM Thank you

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**